### ISSUE TITLE
make install build fails on non-wsl Windows 11 environment

### ISSUE THREAD

**Issue by ekleiner** [gnolang/gno#1060]
### Description

When trying to run `make install` on Windows 11 using non-wsl environment (`cmd`, `Git bash`) - build it fails with the next error:
`..\tm2\pkg\commands\utils.go:44:38: cannot use fd (variable of type syscall.Handle) as int value in argument to term.ReadPassword`

Might not need resolution since Non-WSL environment is not recommended (means, not supported?) according to README.md

### Your environment

* Windows 11 Pro 22621.2134
* master branch b40ac9cda5a5482a4bda7fe12675937a86dba69c

### Steps to reproduce

* Go to gno dir <br />
* Run `make install` <br />
* Observe an error: <br />
```
$ make install
C:/ProgramData/chocolatey/lib/make/tools/install/bin/make.exe --no-print-directory -C ./gno.land        install.gnokey
go install ./cmd/gnokey
# github.com/gnolang/gno/tm2/pkg/commands
..\tm2\pkg\commands\utils.go:44:38: cannot use fd (variable of type syscall.Handle) as int value in argument to term.ReadPassword
make[1]: *** [Makefile:22: install.gnokey] Error 1
make: *** [Makefile:13: install_gnokey] Error 2
```

**Comment by ekleiner** [comment]
Seem to be related to this issue describetd at go:
https://github.com/golang/go/issues/34612

**Comment by thehowl** [comment]
> Seem to be related to issue ...

Slightly - that might be a problem when using `gnokey` at runtime; this is a compilation error.

I followed the issue you linked and adapted the code there for the fix; tell me if #1089 fixes the issue for you. Cheers!
